conda.recipe/build.sh

```diff
@@ -0,0 +1,6 @@
+SRC_DIR=$RECIPE_DIR/..
+cd $SRC_DIR
+npm install .
+npm run build
+python setup.py install
+python -m nbpresent.install --force --enable
```

Comment:
Technically, this line should be put in a post-script file: http://conda.pydata.org/docs/building/build-scripts.html